Goobi/src/de/sub/goobi/helper/tasks/ExportNewspaperBatchTask.java

```diff
@@ -771,6 +771,48 @@ private static Integer positionByRank(List<DocStruct> siblings, String metadataT
         return result;
     }
 
+    /**
+     * Returns the index of the issue to insert between its siblings depending
+     * on its rank. A return value of {@code null} will indicate that no
+     * position could be determined which will cause
+     * {@link DocStruct#addChild(Integer, DocStruct)} to simply append the new
+     * child at the end.
+     *
+     * @param siblings
+     *            brothers and sisters of the child to add
+     * @param rank
+     *            rank of the child to insert
+     * @return the index position to insert the child
+     */
+    private static Integer positionByRank(List<DocStruct> siblings, Integer rank) {
+        if (siblings != null || rank != null) {
+            SIBLINGS: for (int i = 0; i < siblings.size(); i++) {
+                DocStruct aforeborn = siblings.get(i);
```

Comment:
As I did not find any translation: what did `aforeborn` mean or is there some spelling error?

Reply:
children − _Kinder_; siblings − _Geschwister_; aforeborn − _Vorgeborene/r, älterer Bruder/ältere Schwester_
At least now I wonder: Why are the outgoing relations of a tree structure actually called children?